lib/gui/pages/settings/controllers/settings.js

```diff
@@ -74,7 +74,8 @@ module.exports = function(WarningModalService, SettingsModel, AnalyticsService)
 
       AnalyticsService.logEvent('Set setting', {
         setting: name,
-        value: false
+        value: false,
+        dangerous: false
       });
 
       this.model.set(name, false);
```

Comment:
Hmmm, I'm now wondering if the 'prototype' for `this.set` should be updated to `(key, value, dangerous=false)` and then all three of these `'Set setting'` logEvents could be handled just once from within `this.set` ?